container/worker/run_openqa_worker.sh

```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
+# shellcheck disable=SC2012
 set -e
 
 if [[ -z $OPENQA_WORKER_INSTANCE ]]; then
   OPENQA_WORKER_INSTANCE=1
 fi
 
 /root/qemu/kvm-mknod.sh
+
+if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
+  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+  usermod -a -G kvm _openqa-worker
+else
+  echo "Warning: /dev/kvm doesn't exist. If you want to use KVM, run the container with --device=/dev/kvm"
```

Comment:
I don't think this is the right approach as, as stated multiple times, kvm is not a requirement to have a working openQA worker. However, to move forward and as this has no functional impact I will accept it.